lib/sen_tweet/bitfeels/metrics.ex

```diff
@@ -0,0 +1,49 @@
+defmodule SenTweet.Bitfeels.Metrics do
```

Comment:
This is the metrics module we had before but all the state management has been removed.